clang/include/clang/Basic/BuiltinsNVPTX.td

```diff
@@ -580,6 +580,15 @@ def __nvvm_f2bf16_rz : NVPTXBuiltinSMAndPTX<"__bf16(float)", SM_80, PTX70>;
 def __nvvm_f2bf16_rz_relu : NVPTXBuiltinSMAndPTX<"__bf16(float)", SM_80, PTX70>;
 
 def __nvvm_f2tf32_rna : NVPTXBuiltinSMAndPTX<"int32_t(float)", SM_80, PTX70>;
+def __nvvm_f2tf32_rna_satfinite : NVPTXBuiltinSMAndPTX<"int32_t(float)", SM_89, PTX81>;
```

Comment:
Patch summary may need to be updated:
* we're also adding clang builtins, not just LLVM intrinsics.
* we're also adding them for tf32

Perhaps make the summary generic "Added builtins/intrinsics for conversions between new FP types." and include itemized list in the details.